liquid-compiler/src/filter.rs

```diff
@@ -37,8 +37,8 @@ pub trait FilterParametersReflection {
 /// it is necessary to use the `#[filter(...)]`  helper attribute. See documentation on
 /// `liquid-derive` for more information.
 pub trait FilterReflection {
-    fn name(&self) -> &'static str;
-    fn description(&self) -> &'static str;
+    fn name(&self) -> &str;
```

Comment:
I'm surprised, didn't you need to update liquid-derive?

Reply:
Not that I'm aware, everything compile and I can even use liquid in antimoine as it is with this PR (against liquid 0.19.1, not master)

Reply:
https://github.com/cobalt-org/liquid-rust/blob/master/liquid-derive/src/parse_filter/filter_reflection.rs#L31

Weird, Rust must allow you to over-specify the lifetime in trait implementations.

Reply:
Shall I change the one in liquid-derive?

Reply:
If it doesn't require it I'm unsure if its worth it.